docs/usage/faq.md

```diff
@@ -36,6 +36,26 @@ The maintainers don't follow any release schedule or release cadence.
 This means the Mend Renovate App can lag a few hours to a week behind the open source version.
 Major releases of Renovate are held back until the maintainers are reasonably certain it works for most users.
 
+## How can I see which version the Mend Renovate app is using?
+
+Follow these steps to see which version the Mend Renovate app is on:
+
+1. Go to the [Mend Developer Portal](https://developer.mend.io/)
+1. Sign in to the Renovate app with your GitHub or Bitbucket account
+1. Select the app you would like to log in with: Forking Renovate or Renovate
```

Comment:
> There may be a potentially confusing selection for "Select the App", either `RENOVATE` or `FORKING RENOVATE`. Should the FAQ clarify this in this particular question? Or even have this particular dialog be clearer in its own description of "Select the app you'd like to log in with" (a separate PR on a whole separate repo I'm sure)? Really, it could probably be left alone, since in context, "forking renovate" probably doesn't "feel" like the right choice for most users.

Changing the text on the Mend website is something that I can't do. I'll let `@rarkins` nudge the Mend team, if needed.

That screen should:

- Explain/link to the Renovate docs how to see if you're in Forking Mode or the default mode.
- Put the Renovate app on top, Forking Renovate below that. Most users will use the "default" Renovate app.

Reply:
99% of users use Renovate, not Forking Renovate

Reply:
`@rarkins` I'm sorry, but I don't understand what you want me to _do_ in response to your comment.

Do you mean:

1. Mend should move the buttons around so that "Sign in with Renovate" is the top option, and "Forking Renovate" the bottom option.
2. @HonkingGoose should change the text so it only mentions Renovate, as most users will use that?
3. @HonkingGoose should keep the text we have now for this line.
4. Some combination, or something else. :smile:

Reply:
5. Drop this last point altogether